src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts

```diff
@@ -0,0 +1,98 @@
+import SPELLS from 'common/SPELLS';
+import { CLASSES, getClassBySpecId } from 'game/CLASSES';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import SOULBINDS from 'game/shadowlands/SOULBINDS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent, ApplyDebuffEvent } from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+
+interface SpellInfo {
+  id: number;
+  name: string;
+}
+
+const SPELLS_BY_CLASS: { [key: number]: SpellInfo } = {
+  [CLASSES.DEATH_KNIGHT]: SPELLS.SHACKLE_THE_UNWORTHY,
+  [CLASSES.DEMON_HUNTER]: SPELLS.ELYSIAN_DECREE,
+  [CLASSES.DRUID]: SPELLS.KINDRED_SPIRITS,
+  [CLASSES.HUNTER]: SPELLS.RESONATING_ARROW,
+  [CLASSES.MAGE]: SPELLS.RADIANT_SPARK,
+  [CLASSES.MONK]: SPELLS.WEAPONS_OF_ORDER,
+  [CLASSES.PALADIN]: SPELLS.DIVINE_TOLL,
+  [CLASSES.PRIEST]: SPELLS.BOON_OF_THE_ASCENDED,
+  [CLASSES.ROGUE]: SPELLS.ECHOING_REPRIMAND,
+  [CLASSES.SHAMAN]: SPELLS.VESPER_TOTEM,
+  [CLASSES.WARLOCK]: SPELLS.SCOURING_TITHE,
+  [CLASSES.WARRIOR]: SPELLS.SPEAR_OF_BASTION,
+};
+
+const CDR_PER_APPLICATION = 1 / 15;
```

Comment:
If it were me I'd throw in some comments explaining why 1/15 (e.g. 1/15th of the cooldown is the CDR given per application)

Reply:
:+1: